content/docs/conditional-rendering.md

```diff
@@ -35,11 +35,8 @@ function Greeting(props) {
   return <GuestGreeting />;
 }
 
-ReactDOM.render(
-  // Try changing to isLoggedIn={true}:
-  <Greeting isLoggedIn={false} />,
-  document.getElementById('root')
-);
+const root = ReactDOM.createRoot(document.getElementById('root')); 
+root.render(<Greeting isLoggedIn={false} />);
```

Comment:
This is missing the original comment `// Try changing to isLoggedIn={true}:`

Reply:
@eps1lon I have added a comment on the <Greeting /> component. You may review it again. 

Thank you.